Content.Server/Speech/EntitySystems/FrontalLispSystem.cs

```diff
@@ -1,5 +1,6 @@
 using System.Text.RegularExpressions;
 using Content.Server.Speech.Components;
+using Robust.Shared.Random; // Corvax-Localization
```

Comment:
Лучше менять маркер корвакса на наш, а то я по привычке приоритет отдаю визам тогда
```suggestion
using Robust.Shared.Random; // Adventure social anxiety
```